pkg/clustertree/cluster-manager/cluster_controller.go

```diff
@@ -291,15 +291,15 @@ func (c *ClusterController) setupControllers(mgr manager.Manager, cluster *kosmo
 		return fmt.Errorf("error starting podUpstreamReconciler %s: %v", podcontrollers.LeafPodControllerName, err)
 	}
 
-	err := c.setupStorageControllers(mgr, nodes, leafClientset, cluster.Name)
+	err := c.setupStorageControllers(mgr, utils.IsOne2OneMode(cluster), cluster.Name)
 	if err != nil {
```

Comment:
we can use leafModelHandler( AggregationModelHandler DispersionModelHandler)  other than if .... else ...